### PR TITLE
Login and signup page

### PR DESCRIPTION
**Pull request by parulgupta2001**
1. Login as guest functionality added.
2. Signup and login functionality added using redux toolkit

### CODE REVIEW

File: src/frontend/components/left-navbar/Left-navbar.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
+import "./left-navbar.css";
+import { FaFeatherAlt } from "react-icons/fa";
+import { AiFillHome } from "react-icons/ai";
+import { RiHashtag } from "react-icons/ri";
+import { IoNotificationsOutline } from "react-icons/io5";
+import { BiEnvelope } from "react-icons/bi";
+import { FiBookmark } from "react-icons/fi";
+import { CgProfile } from "react-icons/cg";
+
```
Comment (sri-rishi) [inline src/frontend/components/left-navbar/Left-navbar.js:9]: Suggestion: can create separate file for icons and import and export from there

File: src/frontend/pages/login/Login.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
+import { Link, useLocation, useNavigate } from "react-router-dom";
+import { useState } from "react";
+import { useDispatch } from "react-redux";
+import { login } from "../../features/auth-slice";
+import "./login.css";
+
+export function Login() {
+  const [detail, setDetail] = useState({});
```
Comment (sri-rishi) [inline src/frontend/pages/login/Login.js:8]: Suggestion: `detail` can be named as `userDetails` or `loginDetail`

**Review by sri-rishi — APPROVED** [review]
code is clean , just some suggestions which I think can done else feel free to merge

**Review by harshalini — APPROVED** [review]
Code is clean, go ahead to merge the PR

**Pull request merged**